### DISCUSSION TITLE
osu!lazer stopped starting after installing update 2021.619.1

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#13574] [Q&A] [answered]
So today I ran osu! Lazer and installed a new update, nothing happened after restarting even though osu! Lazer was running in task manager. A couple of updates ago, I had no problem launching. I rebooted my computer, reinstalled osu! Lazer, disabled antivirus, but none of that worked for me. What should I do, to fix this problem?

**Comment by Game4all** [comment]
Please attach logs as there are already a few issues about the game crashing upon installing the latest update

they are located at:
- `%AppData%/osu/logs` *(on Windows),*
- `~/.local/share/osu/logs` *(on Linux & macOS).*

**Reply by ghost** [reply]
[database.log](https://github.com/ppy/osu/files/6680363/database.log)
[network.log](https://github.com/ppy/osu/files/6680366/network.log)
[performance.log](https://github.com/ppy/osu/files/6680369/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6680370/runtime.log)
[updater.log](https://github.com/ppy/osu/files/6680371/updater.log)

**Comment by peppy** [comment, marked as answer]
Please install the new hotfix release (without uninstalling your existing one). It should fix your issue.

https://github.com/ppy/osu#releases

**Reply by ghost** [reply]
Thank you very much for the help :)
